deploy/stacks/param_store_stack.py

```diff
@@ -128,7 +146,10 @@ def _get_external_id_value(envname, region):
     except:
         try:
             secrets_client = session.client('secretsmanager', region_name=region)
-            secret_value = secrets_client.get_secret_value(SecretId=secret_id)['SecretString']
+            if secrets_client.describe_secret(SecretId=secret_id):
+                secret_value = SecretValue.secrets_manager(secret_id).unsafe_unwrap()
```

Comment:
minor: `secret_value` could be returned directly here

Reply:
I think it was already resolved